cybersixgill_actionable_alerts/manifest.json

```diff
@@ -0,0 +1,80 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b27feb80-b06f-4200-981a-e91a031d62e6",
+  "app_id": "cybersixgill-actionable-alerts",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor activity in all of our sources regarding key company assets and provide real-time alerts on incoming threats, to create dedicated playbooks",
+    "title": "cybersixgill_actionable_alerts",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Cybersixgill logo",
+        "image_url": "images/transparent.png"
+      },
```

Comment:
```suggestion
```

This can be removed. We will be uploading your logo and displaying it along with your integration on the integrations page.